Chutzpah/Utility/BrowserPathHelper.cs

```diff
@@ -0,0 +1,119 @@
+﻿using System.IO;
+using System.Net.Configuration;
+using Microsoft.Win32;
+
+namespace Chutzpah.Utility
+{
+    public class BrowserPathHelper
+    {
+        private const string BrowserIERegPath = "Software\\Microsoft\\Windows\\CurrentVersion\\App Paths\\IEXPLORE.EXE";
+        private const string BrowserChromeRegPath = "Software\\Microsoft\\Windows\\CurrentVersion\\App Paths\\Chrome.EXE";
+        private const string BrowserFirefoxRegPath = "Software\\Microsoft\\Windows\\CurrentVersion\\App Paths\\Firefox.exe";
+        private const string DefaultBrowserProdIdPath = "Software\\Microsoft\\Windows\\Shell\\Associations\\UrlAssociations\\http\\UserChoice";
+
+        public static string GetBrowserPath(string browserName)
+        {
+            var browserPath = string.Empty;
```

Comment:
I think it wold make sense to wrap this whole thing in a try catch since if forever reason access to registry throws we should just trace this and return empty string